en/patterns/grid.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Grid - Design System Pattern
+_description: The Grid Pattern Symbol represents a full-blown Grid Component.
+_keywords: Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch HTML, Sketch to HTML, Sketch UI kits
+---
+
+## Grid
+
+Use the Grid Pattern symbol as an alternative starting point when designing a tabular data layout. It provides a full-blown contentwise and stylewise Grid Component that comes with all the Grid functionality, provided in the Components Library.
```

Comment:
Make it so "Use any of the Grid Pattern symbols as an alternative..." this implies there is more than what we show and is confirmed by the bullet list at the bottom.